inc/switchtec/switchtec.h

```diff
@@ -355,8 +355,21 @@ switchtec_boot_phase(struct switchtec_dev *dev);
 int switchtec_set_pax_id(struct switchtec_dev *dev, int pax_id);
 int switchtec_echo(struct switchtec_dev *dev, uint32_t input, uint32_t *output);
 int switchtec_hard_reset(struct switchtec_dev *dev);
+
+typedef struct {
+     uint8_t  : 3;
+     uint8_t minor_state : 4;
+     uint8_t major_state : 4;
+     uint8_t  : 2;
+     uint8_t link_rate: 2;
+     uint32_t : 17;
+     uint32_t tstamp: 26;
+     uint8_t : 6;
+}__attribute__((packed)) ltssm_log_data;
+
+int switchtec_ltssm_log(struct switchtec_dev *dev, int port, int *log_count, ltssm_log_data *log_data);
 int switchtec_status(struct switchtec_dev *dev,
-		     struct switchtec_status **status);
+		struct switchtec_status **status);
```

Comment:
I guess it's unintended change.

Reply:
Yes, I was hoping there was a way to reject a change so it simply reverts to the source as the editor may be introducing something

Reply:
Yes, git is good at this, but you should be reviewing and cleaning up before creating a PR.

Before committing you can use `git add --patch` or `git commit --patch` to only select the changes you want to commit.

After committing, you may use `git reset --patch HEAD~1` to stage reverts for specific parts of the previous commit. (You'd then want to `git commit` and `git reset --hard`).